cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,20 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+                self.stdout.write(self.style.SUCCESS(
+                    "Successfully connected to database"
+                ))
+            except OperationalError:
+                self.stdout.write("Database unavailable, waiting 1 second...")
+                time.sleep(1)
```

Comment:
The connection to the database is not actually being tested here. The `connections['default']` just retrieves the connection object, it doesn't check if the connection is available or not. For that, you need to use `db_conn.ensure_connection()`. Also, it's better to limit the number of attempts for connection to avoid an infinite loop in case of connection failure.